libr/asm/p/asm_mips_gnu.c

```diff
@@ -57,6 +57,16 @@ static int disassemble(struct r_asm_t *a, struct r_asm_op_t *op, const ut8 *buf,
 	disasm_obj.endian = !a->big_endian;
 	disasm_obj.fprintf_func = &generic_fprintf_func;
 	disasm_obj.stream = stdout;
+	
+	if (a->cpu && r_str_casecmp (a->cpu, "mips64r2") == 0) {
```

Comment:
Doing all those strcmp for each disasm attempt is unnecessary. Only do that when the asm.cpu changes

Reply:
okay